### PR TITLE
chore(playground): codelens for active connection will inform about default connected database VSCODE-316

### PR DESCRIPTION
**Pull request by himanshusinghs**
## Description
Original issue highlighted in VSCODE-316 (playground should run in database specified in connection string), is not reproducible anymore. It works as expected - if no `use('database')` call is specified in playground and there is a database specified in connection string then the playground will run in the database specified in connection string.

With this PR, we will now additionally highlight, in the playground, the default database that the playground will run in, if there is one specified in connection string.

|Without default database in connection string|With default database in connection string|
|---------------------------------------------|------------------------------------------|
|`mongodb+srv://<credentials>@host`|`mongodb+srv://<credentials>@host/countries`|
|image|image|

We additionally also provide autocomplete for the collection names on the `db` object when connected using a connection string with default database and there is no explicit `use('DB')` call in the playground.

### Checklist
- [x] New tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] I have signed the MongoDB Contributor License Agreement (https://www.mongodb.com/legal/contributor-agreement)

## Motivation and Context

- [ ] Bugfix
- [ ] New feature
- [ ] Dependency update
- [x] Misc

## Types of changes

- [ ] *Backport Needed*
- [ ] Patch (non-breaking change which fixes an issue)
- [ ] Minor (non-breaking change which adds functionality)
- [ ] Major (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/language/mongoDBService.ts [hunk 1]
```diff
@@ -96,6 +97,16 @@ export default class MongoDBService {
...
   }
 
+  get defaultConnectedDB(): string | null {
+    if (!this.connectionString) {
+      return null;
+    }
+    const connectionString = new ConnectionString(this.connectionString);
+    return connectionString.pathname !== '/'
+      ? connectionString.pathname.substring(1)
+      : null;
```
Comment (Anemy) [inline src/language/mongoDBService.ts:107]: We use this same code in `getActiveConnectionDefaultDB`, how do you feel about adding a shared helper somewhere?
I'm thinking then we can remove the `getActiveConnectionDefaultDB` from connection controller and use only the `getActiveConnectionString` to keep the connection controller interface a bit smaller.

**Review by Anemy — APPROVED** [review]
lgtm, one suggestion on reducing duplicated code

File: src/language/mongoDBService.ts (hunk 1 above)
Comment (himanshusinghs) [reply src/language/mongoDBService.ts:107]: yea that sounds about right, i will take care of that.

Comment (himanshusinghs) [reply src/language/mongoDBService.ts]: addressed in [cda24a](https://github.com/mongodb-js/vscode/pull/621/commits/cda24a93c4de52512432be2733da2a168bf20979)

**Review by alenakhineika — APPROVED** [review]

**Pull request merged**